ci/build_python.sh

```diff
@@ -9,13 +9,22 @@ export CMAKE_GENERATOR=Ninja
 
 rapids-print-env
 
+package_name="cuml"
+package_dir="python"
+
+version=$(rapids-generate-version)
+git_commit=$(git rev-parse HEAD)
+
+sed -i "s/__version__ = .*/__version__ = ${version}/g" ${package_dir}/${package_name}/__init__.py
+sed -i "s/__git_commit__ = .*/__git_commit__ = \"${git_commit}\"/g" ${package_dir}/${package_name}/__init__.py
+
 rapids-logger "Begin py build"
 
 CPP_CHANNEL=$(rapids-download-conda-from-s3 cpp)
 
 # TODO: Remove `--no-test` flag once importing on a CPU
 # node works correctly
-rapids-conda-retry mambabuild \
+RAPIDS_PACKAGE_VERSION=${version} rapids-conda-retry mambabuild \
```

Comment:
to keep this file DRY, should we just do `export RAPIDS_PACKAGE_VERSION=${version}` once at the top?

Reply:
Sure can do that.